### PR TITLE
[IMP] theme_*: review s_empowerment occurrences

### PR DESCRIPTION
**Pull request by chgo-odoo**
*: anelusia, artists, avantgarde, aviato, beauty, bewise, bistro, bookstore, buzzy, clean, cobalt, enark, graphene, kea, kiddo, loftspace, monglia, nano, notes, odoo_experts, orchid, paptic, real_estate, treehouse, vehicle, yes, zap

- requires https://github.com/odoo/odoo/pull/175626

task-4094396
Part of task-4077427

### CODE REVIEW

**Review by xlu-odoo — CHANGES_REQUESTED** [review]
Aside from those two comments, content looks great! 🚀

**Review by xlu-odoo — APPROVED** [review]
LGTM! 🚀

File: theme_avantgarde/static/src/img/.DS_Store
Comment (bso-odoo) [inline theme_avantgarde/static/src/img/.DS_Store:1]: It seems a `.DS_Store` file made its way into the repo.

**Review by bso-odoo — COMMENTED** [review]
Is it intentional that zap has no customization ?

**Review by bvr-odoo — APPROVED** [review]

**Comment by bvr-odoo** [comment]
Commit(s) moved into

- https://github.com/odoo/design-themes/pull/919

**Pull request closed without merging**